android/src/main/kotlin/uz/shs/better_player_plus/BetterPlayerPlugin.kt

```diff
@@ -513,39 +546,40 @@ class BetterPlayerPlugin : FlutterPlugin, ActivityAware, MethodCallHandler {
             .hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE)
     }
 
-    private fun enablePictureInPicture(
-        player: BetterPlayer,
-        sourceHintRect: Rect? = null
-    ) {
+    private fun enablePictureInPicture(player: BetterPlayer) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            player.setupMediaSession(flutterState!!.applicationContext)
```

Comment:
will this cause any impact?

Reply:
Không anh ạ chỗ này là code thừa, em check rồi